test/binarta-angular.spec.js

```diff
@@ -2043,6 +2043,19 @@
                     $ctrl.$onInit();
                     expect($ctrl.onSelect).toHaveBeenCalledWith('payment-method');
                 });
+
+                it('should check if the paymentMethodIsAvailable', function () {
```

Comment:
Persoonlijk zie ik weinig nut in deze test. Ik begrijp dat je hem gebruikt om in de volgende test het correcte gedrag te bereiken.

Wat deze test mij nu zegt is dat de controller een isPaymentMethodAvailable functie heeft maar ik zie niet meteen waarom die beschikbaar moet zijn op de controller. Ik denk niet dat we van plan zijn die te gebruiken ergens. Dus lijkt het mij beter om niemand er mee te verleiden.